src/monitoring/mod.rs

```diff
@@ -17,44 +17,44 @@
 #[cfg(feature = "monitoring_prom")]
 mod prometheus;
 
-pub fn increment_rpc_calls_counter() {
-    #[cfg(feature = "monitoring_prom")]
-    prometheus::RPC_CALL_COUNTER.inc();
-}
+// pub fn increment_p2p_msg_unauthenticated_handshake_received_counter() {
```

Comment:
Are we going to add this commented code back at some point?  If not, then let's just delete it (same below).

Reply:
Comments have been removed